client/test/e2e/patient_invoice/invoiceRegsitry.spec.js

```diff
@@ -0,0 +1,166 @@
+/* jshint expr:true */
+/* global element, by, browser */
+const chai = require('chai');
+const expect = chai.expect;
+
+const helpers = require('../shared/helpers');
+helpers.configure(chai);
+
+const InvoiceRegistryPage = require('./invoiceRegistry.page.js');
+const modalPage = require('./modal.page.js');
+const ReceiptModalPage = require('../receipt_modal/receiptModal.page.js');
+
+
+
+describe('Invoice Registry page', function () {
```

Comment:
These are incredible tests!  Excellent!

@IMA-WorldHealth/local-contributors, this an example of very good test coverage.  If all of 2.x has this number of tests, it will be incredibly solid and maintainable!